### ISSUE TITLE
noModule available error after doing grunt uglify.

### ISSUE THREAD

**Issue by kurapatijayaram** [fraywing/textAngular#1141]
Hi, I am getting following error after performing "grunt uglify" task.

Module 'textAngular' is not available! You either misspelled the module name or forgot to load it. If registering a module ensure that you specify the dependencies as the second argument.

Normally its working fine but when I load the uglified version it is the throwing the mentioned bug.

Can you pls help me on this

**Comment by kurapatijayaram** [comment]
got the issue .... there is some problem with minified versions of textAngular, if i include normal js files uglify worked fine.

**Comment by pbassut** [comment]
Yeah. It's a known issue. This will be solved in the next release.

**Comment by goodies4uall** [comment]
These problems have just reappeared after updating to latest version 1.5.12 from version 1.3.7 
The minimized versions look completely different. 
 My gulp uglify task is not minimizing this section properly:
```
.factory("taToolExecuteAction",["$q","$log",function(a,b){
// this must be called on a toolScope or instance
return function(c){void 0!==c&&(this.$editor=function(){return c});var d,e=a.defer(),f=e.promise,g=this.$editor();
```
it gets converted to:
` .factory("taToolExecuteAction",["$q","$log",function(e,t){return function(e){void 0!==e&&(this.$editor=function(){return e});var n,o=s.defer(),i=o.promise,r=this.$editor;`

Note it is failing at s has no function defer(). If properly minimized this would be a.defer(). I left out where the variable a is passed in for simplicity.

**Comment by JoelParke** [comment]
<p>Do you see this with the current release 1.5.13 as well?</p>

**Comment by adan-dev** [comment]
The problem was solved?
